vocs.config.tsx

```diff
@@ -61,34 +61,43 @@ export default defineConfig({
       </>
     );
   },
-  title: "Build with XMTP",
+  title: 'Build with XMTP',
   description:
-    "XMTP is a secure and decentralized protocol for building communication platforms for the next phase of the internet.",
+    'XMTP is a secure and decentralized protocol for building communication platforms for the next phase of the internet.',
   logoUrl: {
-    light: "/logomark-light-purple.png",
-    dark: "/logomark-dark-purple.png",
+    light: '/logomark-light-purple.png',
+    dark: '/logomark-dark-purple.png',
   },
-  iconUrl: "/x-mark-blue-lightmode.png",
+  iconUrl: '/x-mark-blue-lightmode.png',
   topNav: [
-    { text: "Build agents", link: '/agents/get-started/build-an-agent', match: '/agents' },
-    { text: "Build chat apps", link: '/chat-apps/intro/get-started', match: '/chat-apps' },
-    { text: "Protocol", link: '/protocol/overview' },
-    { text: "Network", link: '/network/run-a-node' },
+    {
+      text: 'Build agents',
+      link: '/agents/get-started/build-an-agent',
+      match: '/agents',
+    },
+    {
+      text: 'Build chat apps',
+      link: '/chat-apps/intro/get-started',
+      match: '/chat-apps',
+    },
+    { text: 'Fund agents and apps', link: '/fund-agents-apps/get-started' },
```

Comment:
@jhaaaa "Fund agents and apps" feels a bit out of place to me in the nav. The other items are all high-level intents that kick off journeys ("build an agent" "learn about the protocol" etc) whereas "fund an app or agent" is a downstream step in the app/agent journeys.